### PR TITLE
Navigation and search

### PR DESCRIPTION
**Pull request by ursmal1948**

### CODE REVIEW

**Review by ulyanalakevich — DISMISSED** [review]

**Comment by krzysztof-gronski** [comment]
I see this branch is compared to old version of "main", try re-submit/update pull request and/or "main" branch on your computer

**Comment by krzysztof-gronski** [comment]
fonts and normalize in index.html are duplicated

**Review by krzysztof-gronski — APPROVED** [review]
fonts corrected, stable, every confilct resolved, LGTM!

**Review by krzysztof-gronski — APPROVED** [review]
fonts correction part2, version ready to synchronize, Good Luck!

File: src/Navigation/index.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  Header,
+  PageTitle,
+  List,
+  ListItem,
+  SearchWrapper,
+  StyledNavLink,
+} from "./styled";
+import Search from "../Search";
+
+const Navigation = () => (
```
[the remark this answers is not in the document]
Comment (ursmal1948) [reply src/Navigation/index.js]: already done!

**Review by ulyanalakevich — APPROVED** [review]

**Review by krzysztof-gronski — APPROVED** [review]
LGTM! Go Merge ;)

**Pull request merged**